lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
 
     # Let the XQueue know that we handled the response
     return JsonResponse({'return_code': 0})
+
+
+# pylint: disable=too-many-statements, bad-continuation
+@login_required
+def render_html_view(request):
+    """
+    This view generates an HTML representation of the specified student's certificate
+    If a certificate is not available, we display a "Sorry!" screen instead
+    """
```

Comment:
The idea is that ops would need to go into Django admin, create a new config model with "enabled" set to True.  Anything in the code that enables this feature would load the model (which is cached) and check whether "enabled" is True (it defaults to False).

From my conversations with DevOps, I think they prefer using configuration models like this over the `FEATURES` dictionary, unless you need the feature flag loaded really early (e.g. to enable/disable middleware).